Gruntfile.js

```diff
@@ -122,7 +123,9 @@ grunt.initConfig( {
 			"test/random.html",
 			"test/regex-filter.html",
 			"test/regex-exclude-filter.html",
-			"test/string-filter.html"
+			"test/string-filter.html",
+			"test/warning.html",
+			"test/warning-once.html"
```

Comment:
are are these files? `test/warning.html` and `test/warning-once.html` does not exist.